src/features/game/events/plant.test.ts

```diff
@@ -38,10 +38,34 @@ describe("plant", () => {
     ).toThrow("Field does not exist");
   });
 
+  it("does not plant if water well does not exsit", () => {
```

Comment:
should be `exist` here

Reply:
@mGranTY You are keeping us on our toes lately! Great work 👏